lib/services.js

```diff
@@ -237,6 +236,17 @@ export default {
                 omVal[val] = overrideMetadata[val];
             });
         }
+
+        if (omVal['x-amz-website-redirect-location'] &&
```

Comment:
let's create a validateWebsiteHeader function and call it from initiate MPU, object put and object copy before the meta headers are retrieved.